diff -Nuar rustc-1.31.0-src.orig/src/liblibc/src/unix/mod.rs rustc-1.31.0-src/src/liblibc/src/unix/mod.rs
--- rustc-1.31.0-src.orig/src/liblibc/src/unix/mod.rs	2018-12-05 00:42:58.000000000 +0100
+++ rustc-1.31.0-src/src/liblibc/src/unix/mod.rs	2018-12-15 20:44:21.036100053 +0100
@@ -287,11 +287,7 @@
         // cargo build, don't pull in anything extra as the libstd dep
         // already pulls in all libs.
     } else if #[cfg(target_env = "musl")] {
-        #[cfg_attr(feature = "stdbuild",
-                   link(name = "c", kind = "static",
-                        cfg(target_feature = "crt-static")))]
-        #[cfg_attr(feature = "stdbuild",
-                   link(name = "c", cfg(not(target_feature = "crt-static"))))]
+        #[link(name = "c")]
         extern {}
     } else if #[cfg(target_os = "emscripten")] {
         #[link(name = "c")]
